ckan/tests/logic/test_logic.py

```diff
@@ -25,7 +25,6 @@ def test_check_access_auth_user_obj_is_not_set():
         assert context["auth_user_obj"] is None
 
 
-@pytest.mark.usefixtures("clean_db", "with_request_context")
 def test_check_access_auth_user_obj_is_set():
     user = factories.User()
```

Comment:
we're creating a user here, should we not clean_db in case some other test relies on listing users?